OpenRA.Mods.Common/Traits/Player/EnemyWatcher.cs

```diff
@@ -100,9 +103,11 @@ public void Tick(Actor self)
 
 		void Announce(Actor self, TraitPair<AnnounceOnSeen> announce)
```

Comment:
Since this method is not called per-actor I don't really see the point in creating a trait that must go on all actor types to be called here.